src/test/unit/comparison.test.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import { getEmptyProfile } from '../../profile-logic/data-structures';
+import { mergeProfiles } from '../../profile-logic/comparison';
+
+describe('mergeProfiles function', function() {
+  it('should set interval of merged profile to minimum of all intervals', function() {
+    const profile1 = getEmptyProfile();
+    const profile2 = getEmptyProfile();
+    profile1.meta.interval = 10;
+    profile2.meta.interval = 20;
+
+    const mergedProfile = mergeProfiles([profile1, profile2], []);
+
+    expect(mergedProfile.profile.meta.interval).toEqual(
+      Math.min(profile1.meta.interval, profile2.meta.interval)
```

Comment:
Please just put the expected value (in this case `10`), we don't need to be too smart in tests :)